lib/node/http.js

```diff
@@ -118,11 +118,16 @@ class HTTP extends Server {
       if (!addr)
         addr = this.pool.hosts.address;
 
+      const treeInterval = this.network.names.treeInterval;
+      const blocksUntilNextInterval =
```

Comment:
Add a relative value to match `ns.stats`, it includes both an absolute height and a relative height.

What to call this value? `blocksUntilNextInterval`?

Reply:
- `nextTreeCommitment`?

Reply:
Ended up going with `nextIntervalHeight` and `blocksUntilNextInterval`

Reply:
@tynes how do you feel about `nextTreeRootHeight` and `blocksUntilNextTreeRoot` it is the same length but matches the `treeRoot` attribute already returned.

Reply:
I like calling it those, makes the API more uniform. Just pushed up the change